src/java/org/apache/cassandra/metrics/CASClientWriteRequestMetrics.java

```diff
@@ -40,13 +40,30 @@ public CASClientWriteRequestMetrics(String scope)
     {
         super(scope);
         mutationSize = Metrics.histogram(factory.createMetricName("MutationSizeHistogram"), false);
-        conditionNotMet =  Metrics.counter(factory.createMetricName("ConditionNotMet"));
+        // scope for this metric was changed in 4.0; adding backward compatibility
+        conditionNotMet = Metrics.counter(factory.createMetricName("ConditionNotMet"),
+                                          new ScopeAliasMetricNameFactory().createMetricName("ConditionNotMet"));
     }
 
     public void release()
     {
         super.release();
         Metrics.remove(factory.createMetricName("ConditionNotMet"));
-        Metrics.remove(factory.createMetricName("MutationSizeHistogram"));
+        Metrics.remove(factory.createMetricName("MutationSizeHistogram"),
+                       new ScopeAliasMetricNameFactory().createMetricName("ConditionNotMet"));
```

Comment:
We can if we change it to static method. I saw that for other backward compatibility it was done with second factory again so decided not to touch the method because of compatibility stuff that I hope to remove at some point. But I might have not been right in my judgement

Reply:
Ops, ignore my previous comment, there is a proper utility function that can be used... I think I got confused by the other place I saw it done in that way but that doesn't make sense here really. Pushing the change